test/extended/openstack/local-bd-etcd.go

```diff
@@ -28,9 +28,33 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack] The OpenShift clu
 	var ctx context.Context
 	var dc dynamic.Interface
 	var controlPlaneFlavor string
-	var clusterCPMS objx.Map
 	var clientSet *kubernetes.Clientset
 
+	getControlPlaneMachineSets := func(ctx context.Context, dc dynamic.Interface) ([]objx.Map, error) {
```

Comment:
I have getControlPlaneMAchineSet at cpms.go https://github.com/openshift/openstack-test/pull/124/commits/cb55d18920ac8cd2724122e427f17709d6d45f35#diff-7b12ea0dbd387e5b0d5a4eef6d705422cd7eabd7cd8093be463d17d9eb882841R132 , Please take a look and tell me if you think it should be removed/moved etc

Reply:
excellent, I'll refactor :-)